Updates/1707_DeadCreaturesStandStateOnAggro.sql

```diff
@@ -0,0 +1,35 @@
+-- Update AI of creatures that have bytes1 animation as dead so that they have standing animation in combat
+-- Set AI to EventAI
+UPDATE
+	`creature_template`
+SET
+	`AIName` = 'EventAI'
+WHERE
+	`Entry` IN (2097, 3920, 4510, 5908, 6391, 7643, 12418, 14757);
+DELETE
+FROM
+	`creature_ai_scripts`
+WHERE
+	`creature_id` IN (2097, 3920, 4510, 5908, 6391, 7643, 12418, 14757);
+-- Add on aggro event
+INSERT INTO
+	`creature_ai_scripts` (`id`, `creature_id`, `event_type`, `event_chance`, `action1_type`, `comment`)
+VALUES
+	(209701, 2097, 4, 100, 47, 'Harlo Barnaby: set UnitStandStateType to UNIT_STAND_STATE_STAND'),
+	(392001, 3920, 4, 100, 47, 'Anilia: set UnitStandStateType to UNIT_STAND_STATE_STAND'),
+	(451001, 4510, 4, 100, 47, 'Heralath Fallowbrook: set UnitStandStateType to UNIT_STAND_STATE_STAND'),
```

Comment:
@cala Whom? I see 3 creatures. And again, how do you know this? :)

Reply:
From the comments on the NPC's wowhead page: http://www.wowhead.com/npc=4510/heralath-fallowbrook#comments and also his wiki page. He is dead, and as such does not attack, but is still killable because his "corpse" is required in a quest. Yet, players cannot interact with dead NPCs, so the creature has all the flags required to appear as dead while still being interact-able hence, killable.

However, I only see 1 creature with entry 4510 in the current database. I tested it this afternoon and it is indeed killable and does not fight back.